examples/math/vectorMathExample/src/ofApp.cpp

```diff
@@ -136,10 +136,10 @@ void ofApp::keyPressed  (int key){
 
     switch (key){
         case 'a':
-            speedOfRotation += 0.05f;
+            speedOfRotation += 0.001f;
```

Comment:
Why this change?

Reply:
otherwise the change is too abrupt. it feels better I think